### ISSUE TITLE
Optional momentum/inertia scrolling

### ISSUE THREAD

**Issue by L0rdthri11** [cgeo/cgeo#16145]
### Tell us your idea!

Using the map, my finger sometimes slips (because I am walking, cycling ...) and then the map scrolls , although I just wanted to click.

### What solution would you suggest?

Having the option to turn off momentum/inertia scrolling **while being on the map**, i. e. only scrolling on the map when I touch the screen.

**Comment by Lineflyer** [comment]
It works like all other apps...why should we introduce such a special function. 

Maybe it would be better to have somehting like "always return to my current position" to restore the position after accidental moving.

**Comment by L0rdthri11** [comment]
The "return to my current position" option already exits (thank god!).

Yes, the app works like almost all the other apps, but that's not a good argument, is it?

This type of scrolling is great when going through lists etc. but I **never** need it when looking at the map. Cachers usually look for caches that are more or less close by. Otherwise they use zooming out and in again to go farther away; or they use the search function or use their lists and so on.

I use the "gpx viewer" app. It behaves similar but the momentum (or speed) is smaller, which makes it easier to use.

So maybe an option to set the speed would be another solution.

**Comment by Lineflyer** [comment]
> The "return to my current position" option already exits (thank god!).

Yes, but I was talking about a feature, that allows to willingly or unwillingly scroll the map but upon a timeout return to the current position. 

That is not implemented today but I know it from other apps (like Locus) and it has a benefit also besides accidental scrolling:
You can e.g. scroll around to looks at the surroudings but still the map will just return after you finished looking around.
